winml/api/windows.ai.machinelearning.native.idl

```diff
@@ -35,4 +35,10 @@ interface ILearningModelDeviceFactoryNative : IUnknown
     HRESULT CreateFromD3D12CommandQueue(ID3D12CommandQueue * value, [out] IUnknown ** result);
 };
 
+[uuid(c71e953f-37b4-4564-8658-d8396866db0d), object, local]
+interface ILearningModelSessionOptionsNative : IUnknown
+{
+    HRESULT IntraOpNumThreadsOverride(UINT32 intraOpNumThreads);
```

Comment:
Follow the naming guidelines:
https://osgwiki.com/wiki/Win32_API_Design_Guidelines

VerbNoun

OverrideIntraOpNumThreads(...)

Reply:
Updated.